spec/features/events_spec.rb

```diff
@@ -145,6 +145,19 @@
     end
   end
 
+  scenario "logged in as Organizer I can push the accept all button to accept all applicants" do
+    login(:organizer)
+    @event.max_participants = 3
+    2.times do |i|
+      @pupil = FactoryGirl.create(:profile)
+      @application_letter = FactoryGirl.create(:application_letter, event: @event, user: @pupil.user)
```

Comment:
I dont think they create two applications for of the same. I think the factory creates a new user instance every time a new profile is created, not sure tho.
But maybe there are better ways to create an user than to create a profile and take its user?